main.pm

```diff
@@ -385,6 +389,9 @@ sub load_inst_tests() {
         else {
             loadtest "installation/skip_registration.pm";
         }
+        if (is_sles4sap) {
```

Comment:
please don't. the next time you want to add another step, you have to rewrite the full block